app/routes/flag.routes.js

```diff
@@ -0,0 +1,5 @@
+var flag = require('../controllers/flag.controller');
+
+module.exports = function(app) {
+    app.post('/flag-listing', flag.flagListing);
```

Comment:
Make the route name `'flags/listing-flags'` to better match the route naming convention of REST APIs.